src/main/java/org/example/App.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+
+import java.sql.*;
+
+public class App {
+    public static final String JDBC_DRIVER = "org.postgresql.Driver";
+    static final String DB_URL = "jdbc:postgresql://localhost:5432/mydb";
+    static final String USER = "postgres";
+    static final String PASS = "root";
+
+    public static void main(String[] args) {
+
+        Connection conn = null;
+        PreparedStatement preparedStatement = null;
+        try {
```

Comment:
put this in a separate class and create separate methods for each type of object
addAccommodation, addFair etc